index.html

```diff
@@ -7,7 +7,14 @@
 </head>
 <body>
     
-
+    <h1>ToDoApp</h1>
+    <h5 id="message" hidden>Error Message</h5>
```

Comment:
```suggestion
    <h5 id="error-message" hidden></h5>
```

Reply:
Bhai, here I planned to show all kinds of toaster messages that's why named as message. In this case should I also rename it as "error-message"?